spec/spec_helper.rb

```diff
@@ -25,6 +25,18 @@ def create_user
            oauth_token: "xxx11")
 end
 
+def stub_bad_omniauth
+  OmniAuth.config.test_mode = true
+  OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new({
+    info: {
+      name: "Ryan Asensio"
+    },
+    credentials: {
+      oauth_token: "pizza",
+    }
+  })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.